tools/cravatp_score_and_annotate/cravatp_submit.xml

```diff
@@ -0,0 +1,306 @@
+<tool id="cravatp_submit" name="CRAVAT-P Submit, Intersect, Check, and Retrieve" version="1.0.0">
+    <description>| Submits, checks for, and retrieves data for cancer annotation. Can support proteogenomic proBED inputs for intersection with genomic VCF input.</description>
```

Comment:
Imho the description is too long. The `Can support awesome filetype` is too much description imho. This should go into the help.